src/Jackett.Common/Indexers/NCore.cs

```diff
@@ -208,9 +208,25 @@ public NCore(IIndexerConfigurationService configService, WebClient wc, Logger l,
         
                     else
                     {
-                        Match m = Regex.Match(release.Title, @""+ seasonep + @"\s?$", RegexOptions.IgnoreCase);
                         if (query.MatchQueryStringAND(release.Title, null, seasonep))
                         {
+                            /* For sonnar if the search querry was english the title must be english also so we need to change the Description and Title */
+                            var temp = release.Title;
+                            
+                            // releasedata everithing after Name.S0Xe0X
+                            var releasedata =release.Title.Split(new[] { seasonep }, StringSplitOptions.None)[1].Trim();
```

Comment:
Any chance that this will fail? E.g. releases ending with S0XE0X? If there are such releases I would suggest to avoid blindly accessing the [1] index (would result in an exception) in this case.

Reply:
In that case releasedata[1] will be an empty string.
(Because of StringSplitOptions.None )

Just for safety I check it with this code 
`String test = "testsplit";
var test2 = test.Split(new[] { "split" }, StringSplitOptions.None);
logger.Info("0 is A"+test2[0]+"A");
logger.Info("1 is A" + test2[1] + "A");

06-13 07:34:50 Info 0 is AtestA
06-13 07:34:50 Info 1 is AA`